lib/install.js

```diff
@@ -0,0 +1,16 @@
+const { resolve } = require('path')
+const fse = require('fs-extra')
+
+// helper paths
+const projectRoot = resolve(__dirname, '../../../')
+const assetsPath = resolve(projectRoot, 'assets')
+const quasarAssetsPath = resolve(assetsPath, 'quasar')
+
+/*
+  if this is a fresh install or ~assets/quasar does not already exist in
+  the project root, then create the default stylus files to aid users
+ */
+if (fse.pathExistsSync(assetsPath) && !fse.pathExistsSync(quasarAssetsPath)) {
```

Comment:
Let's use good 'ol `fs` instead of `fs-extra`. No need to require an additional package.